common/src/main/scala/ca/pigscanfly/models/MessageRetrieval.scala

```diff
@@ -0,0 +1,43 @@
+package ca.pigscanfly.models
+
+import io.circe.{Decoder, Encoder}
+import io.circe.generic.semiauto._
+import io.circe.parser._
+
+case class Message(packetId: Int,
+                   deviceType: Int,
+                   deviceId: Int,
+                   deviceName: String,
+                   dataType: Int,
+                   userApplicationId: Int,
+                   len: Int,
+                   data: String,
```

Comment:
this would also be a pb field (base64 encoded into a string)

Reply:
Yes, data is fetched in String type then we will decode it.